rskj-core/src/test/java/co/rsk/peg/BridgeSupportTest.java

```diff
@@ -2098,6 +2099,52 @@ void rskTxWaitingForSignature_uses_updateCollection_rskTxHash_after_rskip_176_ac
         assertEquals(0, provider.getReleaseTransactionSet().getEntries().size());
     }
 
+    @Test
+    void rskTxWaitingForSignature_uses_pegoutCreation_rskTxHash_after_rskip_375_activation() throws IOException {
+        ActivationConfig.ForBlock activations = mock(ActivationConfig.ForBlock.class);
+        when(activations.isActive(ConsensusRule.RSKIP146)).thenReturn(true);
+        when(activations.isActive(ConsensusRule.RSKIP176)).thenReturn(true);
+        when(activations.isActive(ConsensusRule.RSKIP375)).thenReturn(true);
+
+        BridgeConstants spiedBridgeConstants = spy(BridgeRegTestConstants.getInstance());
```

Comment:
Why spy instead of using a mock? Or perhaps it's even better if we use actual BridgeConstants. That way we can test behaviour for the actual values used in testnet/mainnet